src/java.base/share/classes/com/sun/crypto/provider/PKCS12PBECipherCore.java

```diff
@@ -98,7 +97,7 @@ static byte[] derive(char[] chars, byte[] salt, int ic, int n, int type,
         }
         byte[] key = new byte[n];
 
-        if (useNativePBE) {
+        if (useNativePBE && NativeCrypto.isLoaded()) {
```

Comment:
Why is `NativeCrypto.isNativeCryptoSafe()` not used here?

Reply:
Added `NativeCrypto.isNativeCryptoSafe()`.